h2o-core/src/main/java/water/util/VecUtils.java

```diff
@@ -838,4 +837,49 @@ public void map(Chunk c, NewChunk nc) {
     }
   }
 
+  public static long seed(int cidx) {
+    return (0xe031e74f321f7e29L + ((long) cidx << 32L));
+  }
+
+
+  /**
+   * Randomly shuffle randomly a vector from a Frame
+   * */
+  
+  public static class ShuffleVecTask extends MRTask<ShuffleVecTask> {
```

Comment:
Which ShuffleTask is used? There is still more than one implementation. It is confusing. Please keep only one implementation.

Reply:
I removed the others. This is the one I use